datetime.fifthtry.site/index.ftd

```diff
@@ -21,48 +22,60 @@ margin: $ds.spaces.vertical-gap.medium
 -- ds.heading-medium: `Records`
 margin: $ds.spaces.vertical-gap.medium
 
--- ds.heading-small: datetime
+-- ds.heading-small: `datetime`
 -- ds.column:
 align-content: left
-	-- ds.copy-regular: A datetime record is a record that represents a date and time. The record has the following fields:
-	-- ds.copy-regular: - dt: stores the datetime value
+	-- ds.copy-regular: A `datetime` record is a record that represents a date and time. The record has the following fields:
+	-- ds.copy-regular: - `date`: stores the date as an integer in the format YYYYMMDD (e.g., 20250418 for April 18, 2025)
+	-- ds.copy-regular: - `time`: stores the time as an integer in the format HHMMSSmmmnnnnnnnnn (e.g., 123008817000000000 for 12:30:08.817000000)
+	
 	-- ds.code:
 	lang: ftd
 	
 	\-- record datetime:
-	\integer dt:
-	
+	\integer date:
+	\integer time:	
 	-- end: ds.code
 
 -- end: ds.column
 -- ds.heading-medium: `Functions`
 margin: $ds.spaces.vertical-gap.medium
 
--- ds.heading-small: datetime.now(diff) -> datetime.datetime:
+-- ds.heading-small: `datetime.now(diff) -> datetime.datetime`:
 -- ds.column:
 align-content: left
-	-- ds.copy-regular: Returns the time as an i64 unix epoch string with nanoseconds precision in the UTC timezone. Optional parameter of `diff` which returns the time from now `diff` minutes later.
+	-- ds.copy-regular: Returns a `datetime` record with the current date and time in UTC. The date is stored as YYYYMMDD and the time as HHMMSSmmmnnnnnnnnn (to nanosecond precision, nanoseconds padded with zeros). Optional parameter `diff` returns the time from now `diff` minutes later.
 	-- ds.code:
 	lang: ftd
 	
 	\-- datetime.datetime current: $datetime.now()
 	-- end: ds.code
 -- end: ds.column
 
+-- ds.heading-small: `datetime.to-unixtime(dt: datetime) -> integer`:
```

Comment:
Lets split into two functions, `to-unixtime-millis` (should def fit in a number type in JS that it can be `JSON.stringify`-ed. , and `to-unixtime-nanos` (which it currently is).